Demo/Application/Base/AppDelegate.swift

```diff
@@ -64,7 +64,6 @@ import BraintreeCore
         if let preferences = settings.object(forKey: "PreferenceSpecifiers") as? Array<[String: Any]> {
             var defaultsToRegister: [String: Any] = [:]
             preferences.forEach { prefSpecification in
-                print(prefSpecification)
```

Comment:
This was printing out a ton of not very helpful things to the console. Removing this should allow us to more easily debug things printed to the console.